subprojects/packagefiles/libgpiod/man/meson.build

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+if get_option('manpages')
+    help2man_prog = find_program('help2man')
```

Comment:
A classic case where you might use the tri-state nature of features is here:

```meson
help2man_prog = find_program('help2man', required: get_option('manpages'))
if help2man_prog.found()
    ...
endif
```

It will then let you autodetect when manpages can be built, or force them on or off.

Reply:
(This is an observational comment, not a requested change. I don't have a strong opinion here.)

Reply:
> A classic case where you might use the tri-state nature of features is here:

Oh of course. That's trivial enough. I'll rework it.

Reply:
The same applies to doxygen, although...

If you want to avoid the "auto" case building one and not the other (I don't know if it matters, people using "auto" may well be perfectly content to build as much as they can) then you'd need to hoist their find_programs up and set a `have_manpages = foo.found() and bar.found()` boolean.

In the enabled case, if one of help2man/doxygen is found and not the other, then configuring fails anyway, and in the disabled case, neither one is even looked for. This is just a fun thought about the auto case alone.